common/src/main/java/tachyon/master/MasterClient.java

```diff
@@ -112,9 +112,7 @@ public MasterClient(InetSocketAddress masterAddress, ExecutorService executorSer
    * @param length
    * @param checkpointPath
    * @return true if checkpoint is added for the <code>fileId</code> and false otherwise
-   * @throws FileDoesNotExistException
-   * @throws SuspectedFileSizeException
-   * @throws BlockInfoException
+   * @throws IOException
```

Comment:
Could you fill in the rest of the doc as well (ie. describe the parameters and conditions when exceptions will be thrown)?